tests/surrogate_test.py

```diff
@@ -87,9 +119,13 @@ def test_map_function_1d():
     domain = (-8, 12)
     new_domain = (0, 1)
     f = Surrogate(domain, grid)
-    assert f._mapdomain(2, domain, new_domain) == pytest.approx(0.5)
+    g = GradientSurrogate(domain, grid)
+    assert (f._mapdomain(2, domain, new_domain) and
```

Comment:
You do not need to retest functions that are simply inherited from the parent class.